src/Components/Navigation.jsx

```diff
@@ -96,9 +84,9 @@ const Navigation = () => {
                                     page.key !== PageId.AccessCalculator ? <NavButton key={idx} page={page} /> : null
                                 )}
                             </ButtonGroup>
-                            <RedesignButtonPrimary href="/calculator/landing-0" className={classes.navCalc}>
+                            <Button variant="contained" color="secondary" href="/calculator/landing-0">
```

Comment:
I love this! Thanks for adding code to help make our UI components a bit more modular :)